app/views/claims/bank_details.html.erb

```diff
@@ -6,14 +6,14 @@
       <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
 
       <%= form_for current_claim, url: claim_path(current_policy_routing_name) do |form| %>
-        <fieldset class="govuk-fieldset">
+        <fieldset class="govuk-fieldset" aria-describedby="bank_details-hint">
           <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
             <h1 class="govuk-fieldset__heading">
               <%= t("questions.bank_details") %>
             </h1>
           </legend>
 
-          <span class="govuk-hint">
+          <span class="govuk-hint" id="bank_details-hint">
```

Comment:
Hey, I'm not sure this is needed. Screen readers will read the legend when users navigate to the fieldset. If the intention is to get screenreaders to say `Name on the account. The account you want us to send your payment to.` when a user clicks on the banking name field then I think we need to move the aria described by to the `banking-name` form.

Reply:
I think my assumption here would be this hint would describe the whole group, rather than an individual field. I guess in this case, I'd need to add `role="group"` to the fieldset right?

Reply:
Yeah now when you go to an input it will say e.g. “enter bank account detail. Name on the account“.